src/sagemaker/jumpstart/model.py

```diff
@@ -786,6 +791,70 @@ def register_deploy_wrapper(*args, **kwargs):
 
         return model_package
 
+    def display_benchmark_metrics(self):
+        """Display Benchmark Metrics for deployment configs."""
+        config_names = []
+        latencies = []
+        through_puts = []
+        instance_types = []
+        instance_rates = []
+        for deployment_config in self.list_deployment_configs():
+            config_names.append(deployment_config.get("ConfigName"))
+            instance_types.append(deployment_config.get("BenchmarkMetrics").key()[0])
+
+            for benchmark_metric in deployment_config.get("BenchmarkMetrics").values():
+                if benchmark_metric["name"] == "Latency":
+                    latencies.append(benchmark_metric["value"])
+                if benchmark_metric["name"] == "Throughput":
+                    through_puts.append(benchmark_metric["value"])
+                if benchmark_metric["name"] == "Cost":
+                    instance_rates.append(benchmark_metric["value"])
+
+        df = pd.DataFrame(
+            {
+                "Config Name": config_names,
+                "Min. Latency<br>(Ms/Token)": latencies,
+                "Max. Throughput<br>(Tokens/Second)": through_puts,
+                "Instance Type": instance_types,
+                "Instance Rate<br>($/Hour)": instance_rates,
+            }
+        )
+
+        df["Instance Rate<br>($/Hour)"] = df["Instance Rate<br>($/Hour)"].apply(
+            lambda rate: "${0:.2f}".format(float(rate))
+        )
```

Comment:
The instance rate is not available as part of the benchmark metrics data object, we need to use AWS pricing API to query the rate using current region info.

Reply:
can this be a fast follow to this PR?